docs/Installation.md

```diff
@@ -86,8 +86,9 @@ Install, configure and start database engine (and Perl bindings):
 
 ```sh 
 sudo yum install wget 
-wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm -O /tmp/mysql-community-release-el7-5.noarch.rpm
-sudo rpm -ivh /tmp/mysql-community-release-el7-5.noarch.rpm 
+TMPDIR="`mktemp -d`"
+wget -O "$TMPDIR/mysql-community-release-el7-5.noarch.rpm" http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm
+sudo rpm -ivh "$TMPDIR/mysql-community-release-el7-5.noarch.rpm"

```

Comment:
To simplify even more, replace lines 89-91 with

```sh
sudo rpm -ivh http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm
```

Reply:
@sandoche2k, I'm running an installation up to and including Backend on CentOS.

Reply:
Sweet, thanks!